aerospike/metadata.csv

```diff
@@ -586,6 +587,65 @@ aerospike.namespace.latency.write_ops_sec,gauge,,transaction,,Write operations p
+aerospike.namespace.disable_write_dup_res,gauge,,,,"Disables write duplicate resolution for the namespace.",0,aerospike,
+aerospike.namespace.disallow_null_setname,gauge,,,,Enabling this configuration will error out a record write attempt if done without a set name.,0,aerospike,
+aerospike.namespace.data_in_index,gauge,,,,"Optimization in single bin case, will only allow integer or float stored in index space.",0,aerospike,
+aerospike.namespace.evict_hist_buckets,gauge,,,,Number of histogram buckets used for evictions.,0,aerospike,
+aerospike.namespace.evict_tenths_pct,gauge,,,,Maximum 1/10th percentage of objects to be deleted during each iteration of eviction.,0,aerospike,
+aerospike.namespace.enable_benchmarks_write,gauge,,,,Enable histograms for write transactions.,0,aerospike,
+aerospike.namespace.enable_benchmarks_read,gauge,,,,Enable histograms for read transactions.,0,aerospike,
+aerospike.namespace.enable_benchmarks_storage,gauge,,,,Enable histograms for storage access.,0,aerospike,
+aerospike.namespace.enable_benchmarks_udf,gauge,,,,Enable histograms for udf transactions.,0,aerospike,
+aerospike.namespace.enable_benchmarks_udf_sub,gauge,,,,Enable histograms for udf sub transactions.,0,aerospike,
+aerospike.namespace.enable_benchmarks_batch_sub,gauge,,,,Enable histograms for batch sub transactions.,0,aerospike,
+aerospike.namespace.enable_benchmarks_ops_sub,gauge,,,,Enable histograms for ops sub transactions.,0,aerospike,
+aerospike.namespace.enable_hist_proxy,gauge,,,,Enable histograms for proxy transactions.,0,aerospike,
+aerospike.namespace.enable_xdr,gauge,,,,"This controls, at the namespace level, whether digest log entries are being written to the digest log.",0,aerospike,
+aerospike.namespace.allow_xdr_writes,gauge,,,,"Parameter to control whether to accept write transactions originating from an XDR client.",0,aerospike,
+aerospike.namespace.backgroud_scan_max_rps,gauge,,,,"Maximum records per second (rps) allowed for a background scan (i.e. UDF or ops scan).",0,aerospike,
+aerospike.namespace.allow_non_xdr_writes,gauge,,,,"Parameter to control the writes done by a non-XDR client.",0,aerospike,
+aerospike.namespace.ns_forward_xdr_writes,gauge,,,,"This parameter provides fine grained control at namespace level to forward writes that originated from another XDR to the specified destination datacenters (in xdr section).",0,aerospike,
+aerospike.namespace.query_lookup_error,gauge,,,,Number of secondary index query look-up errors.,0,aerospike,
+aerospike.cluster_key,gauge,,,,Randomly generated 64 bit hexadecimal string used to name the last Paxos cluster state agreement.,0,aerospike,
+aerospike.namespace.ns_cluster_size,gauge,,,,Namespace cluster size,0,aerospike,
+aerospike.namespace.xmem_id,gauge,,,,,0,aerospike,
+aerospike.namespace.migrate_rx_instances,gauge,,,,,0,aerospike,
+aerospike.namespace.migrate_tx_instances,gauge,,,,,0,aerospike,
```

Comment:
Could add  `aggregator.assert_metrics_using_metadata(get_metadata_metrics())` to the e2e test ? That will help ensure that the metrics names are correct.

Like this: https://github.com/DataDog/integrations-core/blob/233cb4a7949148834c68148d971263857ad0686a/active_directory/tests/test_active_directory.py#L28

Reply:
We currently assert this in the aerospike e2e: https://github.com/DataDog/integrations-core/blob/master/aerospike/tests/test_aerospike.py#L57

Reply:
Yes, I added it 🤦